### DISCUSSION TITLE
Make {web_result} optional

### DISCUSSION THREAD

**Discussion by skaftanis** [qunash/chatgpt-advanced#120] [Ideas]
Make {web_result} optional in order to be able to create prompts without results from web. There are use cases for that.

**Comment by qunash** [comment]
Hi, thanks for the suggestion.

Just released beta v3.1.7:

- you can now save ~620x more prompt templates than before (~5MB limit vs ~8KB before)
- {web_results} in prompt templates are now optional


See how you can install the beta version: https://github.com/qunash/chatgpt-advanced/tree/dev#manual-installation

Would love to hear your feedback.

**Reply by Eylon12345** [reply]
Thank you very much for sharing the awesome extension and the beta! I installed the v3.1.7 beta, it does not allow me to save a prompt (the default is also unsaved).so, whenever I prompt ChatGPT with web access, I receive an error message because no prompt has been selected.

**Reply by qunash** [reply]
Hi, thanks for the feedback. Should be fixed now in [3.2.0](https://github.com/qunash/chatgpt-advanced/tree/dev/build)

**Comment by qunash** [comment]
The latest version 3.2.4 now supports prompt templates without web results.
